dot/rpc/subscription/websocket.go

```diff
@@ -27,11 +27,13 @@ import (
 	"sync"
 
 	"github.com/ChainSafe/gossamer/dot/rpc/modules"
+	"github.com/ChainSafe/gossamer/dot/types"
+	"github.com/ChainSafe/gossamer/lib/common"
 	log "github.com/ChainSafe/log15"
 	"github.com/gorilla/websocket"
 )
 
-var logger = log.New("pkg", "subscription")
+var logger = log.New("pkg", "rpc")
```

Comment:
maybe `rpc/subscription` as the package?

Reply:
changed.